plugins/login-resources/src/components/ConfirmationSend.svelte

```diff
@@ -18,14 +18,37 @@
   import { getAccount } from '../utils'
   import { onMount } from 'svelte'
 
-  onMount(async () => {
+  const CHECK_INTERVAL = 1000
+
+  async function checkAccountStatus () {
     const account = await getAccount()
     if (account?.confirmed === true) {
       const loc = getCurrentLocation()
       loc.path[1] = 'selectWorkspace'
       loc.path.length = 2
       navigate(loc)
     }
+  }
+
+  let weAreHere = false
+
+  async function check () {
+    try {
+      await checkAccountStatus()
+    } catch (e) {
+      // we should be able to continue from this state
+    }
+    if (weAreHere) {
+      setTimeout(check, CHECK_INTERVAL)
+    }
+  }
+
+  onMount(() => {
+    weAreHere = true
+    check()
+    return () => {
```

Comment:
Why we need this function? I've could not find this behaviour in svelte documentation.

Reply:
It's a convenient way to handle unmount and to clean everything

https://svelte.dev/docs/svelte#onmount

<img width="854" alt="Screenshot 2023-08-11 at 09 02 21" src="https://github.com/hcengineering/anticrm/assets/3119549/446a5a97-c4ff-4946-a6a0-47d49f082bd6">